README.md

```diff
@@ -24,3 +33,96 @@ For the documentation of the module in `README.md`, the following guidelines app
 
 For the short module description (`shortdesc:`), the following guidelines apply:
 - The description should start with a phrase like "The glorb module reticulates splines" or "The tree module can be used to plant trees".
+
+### Post-install Module Config
+
+**Post-install module config** is used to allow local-users to see & change the behavior of the module on booted system, in order to improve local-user experience.
+
+Example of the module which satisfies the requirements & implements this functionality: `default-flatpaks`
+
+#### Config Requirements
+**Requirements** for post-install module configs exist, as not all modules need this functionality.  
+Following conditions for approved post-install module config implementation are:
+
+- **module performs it's functions on booted system**  
+ Modules which are fully utilized in build-time don't need configuration options, as those are already located in `recipe.yml`.
+- **post-install module config can be implemented without affecting reliability of the system**  
+ Module-maintainer needs to carefully select which type of module to implement based on condition above. If a module compromises system reliability when used on booted system, making the module build-time based should be considered. Examples of this are `rpm-ostree` & `akmods` modules, which are better utilized as build-time modules.
+- **module can have additional useful options for configuring**  
+Which can improve local-user experience.
+- **module can strongly collide with local-user's usage pattern with it's default behavior**  
+Example: `default-flatpaks` module can remove a flatpak app, which local-user used daily.
+
+#### Config Format
+
+In order to keep config files easy to read & reliable to parse, standardized `.yml` markup format is used.  
+[`yq`](https://github.com/mikefarah/yq) tool is used to process `.yml` configs in order to reach the desired goal.
+
+#### Config Directory Structure
+
+**System config:**  
+`/usr/share/bluebuild/module-name/config.yml`
+
+**System config** is a post-install module config which is derived from `recipe.yml` module entry. It is placed in this read-only directory location in order to avoid local-users writing to it. System config is purely intended to inform local-users about which modifications are done in `recipe.yml`, so they can potentially proceed with modifications on their own.
```

Comment:
Should make the terms clearer here. IMW this is not a post-install config, as it can't be modified, and has the exact same module configuration as the recipe. It is also unclear from the whether this file should include the serialized-to-yaml module config from the recipe directly, or if it should be the data actually used by the module post-boot.

Reply:
I'm not sure whether the usecase of "Including a module but not doing anythign with it incase someone might want it" makes sense. One could always make a new custom image for that module, but most likely there'd be a way to do the same thing locally without the module, right?

Reply:
I reworded this a bit, tell me if something else needs to be said

Reply:
This is related to the last thread that you opened in 1st point, so I will resolve this.